### PR TITLE
tpm2_dump_capability: add dumping ECC curves

### PR DESCRIPTION
**Pull request by OYTIS**
Add option to dump elliptic curves supported by the device.
Corresponding capability selector is TPM_CAM_ECC_CURVES.

Signed-off-by: Anton Gerasimov <anton@advancedtelematic.com>

### CODE REVIEW

File: tools/tpm2_dump_capability.c [hunk 1]
```diff
@@ -508,6 +557,7 @@ dump_command_attr_array (TPMA_CC     command_attributes[],
     for (i = 0; i < count; ++i)
         dump_command_attrs (command_attributes [i]);
 }
+
 /*
```
Comment (martinezjavier) [inline tools/tpm2_dump_capability.c]: Unrelated new line added.

File: tools/tpm2_dump_capability.c [hunk 2]
```diff
@@ -562,6 +612,10 @@ static int dump_tpm_capability (TPMU_CAPABILITIES *capabilities) {
         dump_algorithms (capabilities->algorithms.algProperties,
                          capabilities->algorithms.count);
         break;
+    case TPM_CAP_ECC_CURVES:
+	dump_ecc_curves (capabilities->eccCurves.eccCurves,
+                         capabilities->eccCurves.count);
+	break;
```
Comment (martinezjavier) [inline tools/tpm2_dump_capability.c]: I think it would be a good idea to keep the capabilities sorted according to 6.12 TPM_CAP (Capabilities) table in Trusted Platform Module Library part 2.

Rest of the patch looks good, thank a lot for proposing it.

Comment (OYTIS) [reply tools/tpm2_dump_capability.c]: Sorry I don't quite get it. The previous sorting [PROPERTIES, ALGS, COMMANDS] didn't correspond 6.12 as well.

Comment (martinezjavier) [reply tools/tpm2_dump_capability.c]: Indeed and that's not your fault, but I meant that we should try to keep it sorted from now on (since is quite likely that more properties will be added the future).

We can later move TPM_CAP_TPM_PROPERTIES to the end to match the table order.

Comment (OYTIS) [reply tools/tpm2_dump_capability.c]: OK, this makes sense.
